### PR TITLE
main/asciidoc: update url and modernize

### PR DESCRIPTION
**Pull request by tmpfile**

### CODE REVIEW

File: main/asciidoc/APKBUILD [hunk 1]
```diff
@@ -1,44 +1,40 @@
...
+makedepends="autoconf"
 subpackages="$pkgname-doc $pkgname-vim::noarch"
-source="http://sourceforge.net/projects/asciidoc/files/asciidoc/$pkgver/asciidoc-$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/asciidoc/asciidoc/archive/$pkgver.tar.gz"
 
-builddir="$srcdir"/$pkgname-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
 build() {
 	cd "$builddir"
+	autoconf
```
Comment (fabled) [inline main/asciidoc/APKBUILD]: Running `autoconf` is usually done in `prepare()`. Care to change this?

Comment (tmpfile) [reply main/asciidoc/APKBUILD]: I fixed it.

File: main/asciidoc/APKBUILD [hunk 2]
```diff
@@ -21,24 +25,20 @@ build() {
...
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 }
 
 vim() {
-	depends=""
```
Comment (fabled) [inline main/asciidoc/APKBUILD]: Just spotted this. Resetting depends is required. Please don't remove this. After this it looks good to go for me.

Comment (tmpfile) [reply main/asciidoc/APKBUILD]: Fixed

**Pull request closed without merging**